mcodingbot/plugins/python.py

```diff
@@ -46,4 +66,28 @@ async def on_message(event: hikari.MessageCreateEvent) -> None:
         for pep_number in pep_refs
     )
 
-    await event.message.respond(pep_links_message, reply=True)
+    await event.message.respond(pep_links_message, reply=True, component=get_dismiss_button(event.author.id))
+
+
+@plugin.include
+@crescent.event
+async def on_interaction(event: hikari.InteractionCreateEvent) -> None:
+    inter = event.interaction
+
+    if (
+        not isinstance(inter, hikari.ComponentInteraction)
+        or DISMISS_BUTTON_ID not in inter.custom_id
+    ):
+        return
```

Comment:
De Morgan's laws my beloved
```suggestion
    if not (
        isinstance(inter, hikari.ComponentInteraction)
        and DISMISS_BUTTON_ID in inter.custom_id
    ):
        return
```